src/content/en/tools/chrome-devtools/accessibility/navigation.md

```diff
@@ -0,0 +1,348 @@
+the field will be the tag name of the element that is selected in the DOM tree.
+You may type any class name you want into this field and then press tab to
+assign CSS properties to it.
+
+### Computed tab
+
+With focus on the [**Computed**
+tab](/web/tools/chrome-devtools/css/reference#computed), press `Tab` to move
+focus inside and explore its contents.
+
+Within the Computed tab there are controls for exploring which CSS properties
+are actually applied to an element in order of specificity.
+
+#### Explore all computed styles
+
+Press `Tab` until you reach the collection of computed styles. These are
+presented as an ARIA `tree`. Expanding a listbox will reveal which CSS selectors
+are applying the computed style. These selectors are organized by specificity.
+
+A screen reader will announce the computed value, which CSS selector is
+currently matching, the filename of the stylesheet that contains the selector,
+and the line number for the selector.
+
+#### Known issues
+
+- If you use the filter box you can no longer inspect styles.
+
+### Event listeners tab
+
+From within the **Elements** panel you may inspect the event listeners applied
+to an element using the **Event Listeners** tab. // TODO: Is this tab
+documented?
+
+With focus on the Styles tab, press the `Right Arrow` to navigate to the Event
+Listeners tab.
+
+#### Explore event listeners
+
+Event listeners are presented as an ARIA `tree`. You may use the arrow keys to
+navigate them. A screen reader will announce the name of the DOM object that the
+event listener is attached to, as well as the file name where the event listener
+is defined and its line number.
+
+### Accessibility tab
+
+With focus on the [**Accessibility**
+tab](/web/tools/chrome-devtools/css/reference#computed), press `Tab` to move
+focus inside and explore its contents.
+
+Within the Accessibility tab there are controls for exploring the accessibility
+tree, the ARIA attributes applied to an element, and its computed accessibility
+properties.
+
+#### Accessibility tree
+
+The accessibility tree is presented as an ARIA `tree` where each `treeitem`
+corresponds to an element in the DOM.
+
+The accessibility tree will announce the computed role for the selected node.
+Generic elements like `div` and `span` will be announced as "GenericContainer"
+in the tree.
+
+Use the arrow keys to traverse the tree and explore parent-child relationships.
+
+#### Known issues
+
+- The type of ARIA tree used by the Accessibility panel may not be properly
+  exposed in Chrome for macOS screen readers like VoiceOver. Please subscribe to
+  [the chromium tracking
+  bug](https://bugs.chromium.org/p/chromium/issues/detail?id=868480) for more
+  information.
+- The ARIA Attributes and Computed Properties sections are marked up as ARIA
+  trees, but they do not currently have focus managmenet so they are not
+  keyboard operable.
+
+## Audits panel
+
+The **Audits** panel let's you run a series of tests against a site to check
```

Comment:
@kaycebasques 	do you know if there is an official doc for the audits panel? If so i was going to link to it here.